functions/__tests__/config.test.ts

```diff
@@ -128,4 +128,42 @@ describe('extensions config', () => {
       })
     })
   })
+
+  // SEARCHABLE_FIELDS
+  describe('Test searchableFields parameter', () => {
+    test('param exists', () => {
+      const extensionParam = extensionParams['SEARCHABLE_FIELDS']
+      expect(extensionParam).toMatchSnapshot()
```

Comment:
It's just a tool for when I make my tests and also to see what I tested.
But I can remove it

Reply:
I'm not sure I understand what it does

Reply:
It creates a snapshot file when you run the test the first time and check when you re-run the test that the parameter hasn't changed. So it won't work on the CI it's just for the local test or we can add the snapshot reference file on the repository to be sure the extension parameter won't be modified by mistake.
I use it in two places. The first is on my extension setting in the `config.test.ts` to check that I haven't changed my settings (extension parameters). And the second is on the environment in `util.test.ts` because I have tried many ways to change my environment variables at runtime.